starwars/src/components/CharacterList.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./StarWars.css";
+import Characters from "./Characters";
+
+// This is grabbing the information passed through my PROP and mapping the information(array).
+const CharacterList = props => {
+  return (
+    <div className="character-container">
+      {props.CharacterInfo.map(item => (
+        <Characters objInfo={item} />

```

Comment:
I'm not sold on your naming conventions, but you can name everything whatever you want